home/management/commands/import_json_content_turn.py

```diff
@@ -0,0 +1,94 @@
+                try:
+                    im = Image.objects.get(title=title).id
+                except Exception:
+                    http_res = requests.get(message["attachment_uri"])
+                    image_file = ImageFile(BytesIO(http_res.content), name=title)
+                    image = Image(title=title, file=image_file)
+                    image.save()
+                    im = image.id
+
+                block = blocks.StructBlock(
+                    [("message", blocks.TextBlock()), ("image", ImageChooserBlock())]
+                )
+                block_value = block.to_python(
+                    {"message": message["answer"], "image": im}
+                )
+                return block_value
+            else:
+                block = blocks.StructBlock(
+                    [
+                        ("message", blocks.TextBlock()),
+                    ]
+                )
+                block_value = block.to_python({"message": message["answer"]})
+                return block_value
+
+        #
+        # def create_tags(tags, page):
+        #     for tag in tags:
+        #         created_tag, _ = Tag.objects.get_or_create(name=tag)
+        #         page.tags.add(created_tag)
+
+        path = Path(options["path"])
+
+        ContentPage.objects.all().delete()
+        with path.open() as json_file:
+            data = json.load(json_file)
+
+            for message in data["data"]:
+                try:
+                    title_list = message["question"].strip().split(" ")
+                    language = title_list[-1]
+                    language = language[1:-1]
+                    home_page = HomePage.objects.get(title__icontains=language)
+                    just_title = " ".join(title_list[0:-1])
+
+                    contentpage = ContentPage(
+                        title=message["question"],
+                        whatsapp_title=message["question"],
+                        whatsapp_body=[("Whatsapp_Message", get_body(message))],
+                        locale=home_page.locale,
+                    )
+                    # create_tags(article["tags"], contentpage)
+                    home_page.add_child(instance=contentpage)
+                    contentpage.save_revision().publish()
+
+                    translations = ContentPage.objects.filter(
+                        title__icontains=just_title
+                    )
+                    if translations:
+                        translation = translations.first()
+                        if translation.pk != contentpage.pk:
+                            key = translation.translation_key
+                            contentpage.translation_key = key
+                            contentpage.save_revision().publish()
+                except Exception as e:
+                    print(e)
+
+            self.stdout.write(self.style.SUCCESS("Successfully imported Content Pages"))
```

Comment:
If we're keeping this, then we should have tests for it, to ensure that it actually works correctly.

Reply:
So the plan was to revert that PR which removed it and then in a seperate pr, push tests for it. 
I can go ahead and add the tests in this pr if that's the case.